packages/block-library/src/post-date/block.json

```diff
@@ -2,7 +2,7 @@
 	"$schema": "https://schemas.wp.org/trunk/block.json",
 	"apiVersion": 3,
 	"name": "core/post-date",
-	"title": "Post Date",
```

Comment:
I think just date is confusing as it could just as well be the current date. Maybe this should just be called Published Date

Reply:
Hm, maybe so. 🤔

Reply:
Calling it "published date" would help separate it from the "modified date" variation.

Reply:
It's actually a bit weird for me to see modified date as a variation in the inserter. I'd expect it to be a single date block and for modified to be a property there.